dencam/gui.py

```diff
@@ -103,14 +103,8 @@ def _update(self):
 
         self.recorder.update_timestamp()
 
-        if self.state.value == 1:
-            self.show_frame('NetworkPage')
-        elif self.state.value == 2:
-            self.show_frame('RecordingPage')
-        elif self.state.value == 3:
-            self.show_frame('SolarPage')
-        elif self.state.value == 4:
-            self.show_frame('BlankPage')
+        if self.state.value >- 1 and self.state.value <= 4:
```

Comment:
Did you mean to have

```
>- 1 
```

The following seems like more common syntax:

```
>= 0 
```

or >= 1 if that is what you actually wanted